js_modules/dagit/packages/core/src/instance/backfill/__tests__/BackfillPage.test.tsx

```diff
@@ -135,25 +155,32 @@ describe('BackfillPage', () => {
       ),
     );
 
-    await waitFor(() => getByText('assetA'));
+    await waitFor(() => {
+      // Check if the loaded content is displayed
+      expect(screen.getByText('Jan 1, 1970, 12:16:40 AM')).toBeVisible();
```

Comment:
pleasantly surprised this isn't timezone sensitive 👀

Reply:
🙄